### PR TITLE
Youth Endpoints

### PR DESCRIPTION
**Pull request by b-walton**

### CODE REVIEW

**Review by samshefnie — COMMENTED** [review]
looks good!

File: src/controllers/youth.controller.ts [hunk 1]
```diff
@@ -1 +1,275 @@
...
+      { firebaseUID: firebaseUID }, //filter by ID
+      { $set: { active: false } }, //deactivate youth
+      { new: true }, // return new obj
+    );
+    if (!updatedYouth) {
+      throw new HttpError(
+        HttpStatus.NOT_FOUND,
+        "No youth with id: " + firebaseUID,
+      );
+    } else if (updatedYouth.get("active") == true) {
```
Comment (lunstb) [inline src/controllers/youth.controller.ts]: tiny nit but why == true

File: src/controllers/youth.controller.ts [hunk 2]
```diff
@@ -1 +1,275 @@
...
+      { firebaseUID: firebaseUID }, //filter by ID
+      { $set: { active: true } }, //activate youth
+      { new: true }, // return new obj
+    );
+    if (!updatedYouth) {
+      throw new HttpError(
+        HttpStatus.NOT_FOUND,
+        "No youth with id: " + firebaseUID,
+      );
+    } else if (updatedYouth.get("active") == false) {
```
Comment (lunstb) [inline src/controllers/youth.controller.ts]: same tiny nit here

**Review by lunstb — APPROVED** [review]
lgtm. left tiny nits but good. either address the little nits or ignore them if you feel differently.

**Pull request merged**